requirements.txt

```diff
@@ -1,5 +1,5 @@
-nupic==0.5.7.dev0
-nupic.bindings==0.4.13.dev0
+nupic
+nupic.bindings
```

Comment:
Ok, but this codebase will NOT work with some older versions of NuPIC. I expect to keep it aligned with tip of master, so it continue to be bleeding edge.